tests/json/schema-1-response/input.py

```diff
@@ -0,0 +1,87 @@
+#! test QC_JSON Schema for response properties
+
+import pytest
+import psi4
+import numpy as np
+import json
+import os
+from distutils import dir_util
+
+
+@pytest.fixture
+def datadir(tmpdir, request):
+    """
+    from: https://stackoverflow.com/a/29631801
+    Fixture responsible for searching a folder with the same name of test
+    module and, if available, moving all contents to a temporary directory so
+    tests can use them freely.
+    """
+    filename = request.module.__file__
+    test_dir, _ = os.path.splitext(filename)
+
+    if os.path.isdir(test_dir):
+        dir_util.copy_tree(test_dir, str(tmpdir))
+
+    return tmpdir
+
+
+def test_response(datadir):
```

Comment:
is this test definitely running? it's using pytest fixtures and the `def test_*` collection pattern, but since it's in `tests/json/` rather than `tests/pytests/`, it's being executed by ctest and since everything's behind a `def`, I'm concerned it's a free pass. Maybe pytest is more powerful than I know, so just checking.

Reply:
good catch. CI doesn't run any of these json tests besides energy and gradient. but if you do force test these, the function inside `schema-1-response/input.py` does not get run. I can can pull it out of the function def, axe the fixtures, and put the reference data into the input directly instead of loading it from file. would that be fine?

Reply:
that would be fine. or you can move the file and ref file to tests/pytests/. either way

Reply:
done. to make sure it was _actually_ testing, I tried putting a "bad" value in and ran the test, and it failed as expected.